app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def login
+    user = User.find_by(name: params[:name])
```

Comment:
Please make sure that a user can log in with an email and password.
You can change the line to user = User.find_by(email: params[:email])